domain-management/src/main/java/org/jboss/as/domain/management/audit/SyslogAuditLogHandlerResourceDefinition.java

```diff
@@ -277,6 +281,35 @@ private static String resolveAppName(OperationContext context, ModelNode appName
         return DEFAULT_APP_NAME_IF_NOT_A_PRODUCT;
     }
 
+    static class SyslogAuditLogHandlerRemoveHandler extends AbstractRemoveStepHandler {
+
+        private final ManagedAuditLogger auditLogger;
+
+        SyslogAuditLogHandlerRemoveHandler(ManagedAuditLogger auditLogger) {
+            this.auditLogger = auditLogger;
+        }
+
+        @Override
+        protected boolean requiresRuntime(OperationContext context) {
+            return true;
+        }
+
+        @Override
+        protected void performRuntime(OperationContext context, ModelNode operation, ModelNode model) throws OperationFailedException {
+            String name = Util.getNameFromAddress(operation.require(OP_ADDR));
```

Comment:
No need to change this but this should be String name = context.getCurrentAddressValue();